terraform/main.tf

```diff
@@ -38,7 +38,7 @@ locals {
   }
   proxies = {
     for name, instance in local.provisioner_pairs :
-    instance.prefix => true
+    instance.prefix => true...
```

Comment:
this was needed for when servers have a shared prefix? Not sure how others are getting around this. Maybe we could use a list or set instead of a map? This `...` does a group-by-key which I think is also fine

Reply:
I suggest refactoring this to a set.
The `instance.prefix => true` was/is already a hack. Adding a `...` group-by, is maybe too hacky?

Reply:
I mean. If the hack still works I'm ambivalent. If you wanna submit a followup to change this to a set I'm 👍 on that